src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c

```diff
@@ -848,34 +848,35 @@ void xclmgmt_mailbox_srv(void *arg, void *data, size_t len,
 
 		/* Passthrough Virtualization feature configuration */
 		if (xocl_passthrough_virtualization_on(lro)) {
-			if (!iommu_present(&pci_bus_type)) {
-				p2p_bar_addr = mb_p2p->p2p_bar_addr;
-				p2p_bar_len = mb_p2p->p2p_bar_len;
-				mgmt_info(lro, "got the p2p bar addr = %lld\n", p2p_bar_addr);
-				mgmt_info(lro, "got the p2p bar len = %lld\n", p2p_bar_len);
-				if (!p2p_bar_addr) {
-					pci_write_config_byte(pdev, 0x188, 0x0);
-					ret = 0;
-					(void) xocl_peer_response(lro, req->req, msgid, &ret,
+			p2p_bar_addr = mb_p2p->p2p_bar_addr;
+			p2p_bar_len = mb_p2p->p2p_bar_len;
+			mgmt_info(lro, "got the p2p bar addr = %lld\n", p2p_bar_addr);
+			mgmt_info(lro, "got the p2p bar len = %lld\n", p2p_bar_len);
+			if (!p2p_bar_addr) {
+				pci_write_config_byte(pdev, 0x188, 0x0);
+				pci_write_config_dword(pdev, 0x190, 0x0);
+				pci_write_config_dword(pdev, 0x194, 0x0);
+				pci_write_config_dword(pdev, 0x18C, 0x0);
```

Comment:
can we add some comments and change numbers like 0x188 to some meaningful macro names here? the current code is not readable and will be broken for sure in the future, if we keep it this way.